tests/plugin.test.ts

```diff
@@ -27,7 +27,19 @@ describe("Purchases", () => {
       null,
       "PurchasesPlugin",
       "configure",
-      ["api_key", "app_user_id", true, undefined]
+      ["api_key", "app_user_id", true, null, false]
+    );
+  });
+
+  it("configure fires PurchasesPlugin with the correct arguments when using Amazon", () => {
+    Purchases.configure("api_key", "app_user_id", true, undefined, true);
```

Comment:
It's sad this is how it needs to be called if someone wants `useAmazon` to be `true` 

It makes me reconsidering adding a function that accepts `PurchasesConfiguration` 😢